homeassistant/components/tplink/coordinator.py

```diff
@@ -60,6 +61,9 @@ def __init__(
                 hass, _LOGGER, cooldown=REQUEST_REFRESH_DELAY, immediate=False
             ),
         )
+        self._previous_child_device_ids = {child.device_id for child in device.children}
+        self.removed_child_device_ids: set[str] = set()
```

Comment:
```suggestion
        self._removed_child_device_ids: set[str] = set()
```
Should this also be marked as private?

Reply:
This is accessed from `entity` to ensure that removed devices will be re-added